cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -25,9 +25,9 @@ This section comprises a glossary of all the keywords -- grouped by category and
 |<<query-call-subquery, CALL {...}>>        | Reading/Writing   | Evaluates a subquery, typically used for post-union processing or aggregations.
 |<<query-create, CREATE>>                    | Writing     |  Create nodes and relationships.
 |<<administration-constraints-syntax, CREATE CONSTRAINT [existence] [IF NOT EXISTS] ON (n:Label) ASSERT n.property IS NOT NULL>>  | Schema   | Create a constraint ensuring that all nodes with a particular label have a certain property.
-|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
+|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
```

Comment:
This sentence could keep the extra _that_ : "...that ensures that all nodes.."

Reply:
```suggestion
|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
```